app/views/section1/ducr_entry.scala.html

```diff
@@ -83,8 +82,6 @@
             content = HtmlContent(expanderContent)
         ))
 
-        @tariffExpander(Ducr, request.declarationType)(messages, appConfig)
```

Comment:
The injection of `AppConfig` at L41, as well as the `@import`, should be removed